corgidrp/data.py

```diff
@@ -1317,6 +1333,53 @@ def __init__(self, data_or_filepath, pri_hdr=None, ext_hdr=None, input_dataset=N
         if 'DATATYPE' not in self.ext_hdr or self.ext_hdr['DATATYPE'] != 'AstrometricCalibration':
             raise ValueError("File that was loaded was not an AstrometricCalibration file.")    
         
+class TrapCalibration(Image):
+    """
+
+    Class for data related to charge traps that cause charge transfer inefficiency. 
+    The calibrationi is generated by trap-pumped data. 
+
+    The format will be [n,10], where each entry will have: 
+    [row, column, sub-electrode location, index numnber of trap at this pixel/electrode, 
+    capture time constant, maximum amplitude of the dipole, energy level of hole, 
+    cross section for holes, R^2 value of fit, release time constant]
+
+     Args:
+        data_or_filepath (str or np.array): either the filepath to the FITS file to read in OR the 2D image data
+        pri_hdr (astropy.io.fits.Header): the primary header (required only if raw 2D data is passed in)
+        ext_hdr (astropy.io.fits.Header): the image extension header (required only if raw 2D data is passed in)
+        input_dataset (corgidrp.data.Dataset): the Image files combined together to make this bad pixel map (required only if raw 2D data is passed in)
```

Comment:
"bad pixel map" --> "trap calibration"

Reply:
And I see no `err_hdr`, and I think that's fine for this particular calibration.  It would amount to running an iterative trap correction algorithm for the central tau and cap values, and then multiple other times for the upper and lower bounds of these, and in practice, we could probably get an err on the observation frames we CTI-correct from the correction process itself apart from considering the error in tau and cap.

Reply:
On second thought, I think in general those uncertainties the II&T function returns would be nice to have in an `err_hdr`, but as the 9/9 deadline is approaching, I'm okay keeping that as an enhancement in an issue.